### PR TITLE
[CharConversion] Implement logic

### PR DESCRIPTION
**Pull request by plotnikov01**

### CODE REVIEW

File: src/char_conv.c [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+int main() 
+{
+	char string[] = "Hello World!";
+
+	int i = 0;
+
+	while(string[i] != '\0') {
+		printf("Ascii value is %d \n", string[i]);
```
Comment (MLGMAG) [inline src/char_conv.c:10]: Extract `string[i]` into separate variable.

File: src/char_conv.c [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+int main() 
+{
+	char string[] = "Hello World!";
+
+	int i = 0;
+
+	while(string[i] != '\0') {
```
Comment (MLGMAG) [inline src/char_conv.c:9]: Extract `\0` into constant

File: src/char_conv.c [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+int main()
```
Comment (MLGMAG) [inline src/char_conv.c:3]: Change to `int main() {`

**Review by MLGMAG — CHANGES_REQUESTED** [review]

**Comment by MLGMAG** [comment]
Commit message should be: `[topic] description`

Change commit message to `[CharConversion] Implement logic`

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/char_conv.c (hunk 1 above)
Comment (MLGMAG) [reply src/char_conv.c]: Still present

Comment (plotnikov01) [reply src/char_conv.c]: done

**Review by MLGMAG — APPROVED** [review]

**Pull request merged**